src/code/z_camera.c

```diff
@@ -476,7 +476,7 @@ PlayerMeleeWeaponState func_800CBAAC(Camera* camera) {
     }
 }
 
-s32 Camera_GetFocalActorPos(Vec3f* dst, Camera* camera) {
+Vec3f* Camera_GetFocalActorPos(Vec3f* dst, Camera* camera) {
     PosRot focalPosRot;
     Actor* focalActor = camera->focalActor;
```

Comment:
Amazed I never caught that

Reply:
Stuff like this is so easy to miss, this is why I want this warning to be enabled